output/list.go

```diff
@@ -33,26 +33,28 @@ var green = color.New(color.FgGreen).SprintFunc()
 var yellow = color.New(color.FgYellow).SprintFunc()
 var red = color.New(color.FgRed).SprintFunc()
 
-// List is a ReporterFunc that produces a verbose list of results
-var List ReporterFunc = func(config *plan.Config, tests <-chan execute.TestResponse) Summary {
-	var summary Summary
-	for test := range tests {
-		for _, result := range test.Results {
-			var statStr string
-			switch result.Status {
-			case execute.Success:
-				statStr = green("✓")
-				summary.NumSuccess++
-			case execute.Skipped:
-				statStr = yellow("S")
-				summary.NumSkipped++
-			default:
-				statStr = red("F")
-				summary.Failed = true
-				summary.NumFail++
-			}
-			fmt.Printf("%v - %v - %v\n", statStr, test.TestCase, result.Output)
+type List struct {
+}
```

Comment:
really nitty nit: close empty structs on the same line with no space between the keyword and the `{`.

```go
type List struct{}
```

---

Also note that for stateless implementations of an interface, it's acceptable to provide a global instance:

```go
type list struct{}

func (l *list) ...

var List Reporter = &list{}
```

Then you can just use `output.List` as the reporter instead of `&output.List{}`.